app/Http/Controllers/PublicProfileController.php

```diff
@@ -29,6 +29,23 @@ public function index()
             ->with('speakers', $users);
     }
 
+    public function search(Request $request)
+    {
+        $users = User::search($request->search_query)
+            ->orderBy('name', 'asc')->get();
+
+        // Since Scout searches can only perform rudimentary where clauses,
+        // we must filter search results to only validly public profiles.
+        $filteredUsers = $users->filter(function ($user) {
```

Comment:
Per the Scout documentation:

> Currently, these clauses only support basic numeric equality checks, and are primarily useful for scoping search queries by a tenant ID.

Reading through Scout documentation and code, there is no way that I see to index a profile since the profile is a view and not a model. I will think about this one a bit more to see if there's an option I haven't thought of yet. For now, I think filtering the results is our best option.